Add feature to show hidden apps on wishlist

candela97 opened the pull request:
An alternative idea to resolve #407. Rather than manually inserting rows into the wishlist, display hidden apps in a separate dialog instead, this way we can include other useful links (only add discussions/ITAD/SteamDB links for now) along with the app.

Some thoughts:
1. Hidden apps count counts the whole wishlist, and will not change after filtering. Maybe make it a separate section in the stats popup?
2. Styling in the dialog. Usable right now but has a lot of empty space.
3. Fade out/blur removed rows instead of hiding them? Helps users keep track if there're a lot of apps.

MxtOUT commented:
- The amount of hidden apps is hidden (`display: none`), so I can't open the dialog normally
- It is not apparent that the number is clickable unless you hover over it, some better indication is required (this should also resolve your first point)
- The texts "View on ITAD" and "View on SteamDB" should be replaced by their respective icons
- Should the link to the community page go to the homepage or to the discussions?
- Ad 3: I think it's the other way around, since you actually free up space by removing the row

candela97 commented:
> * The amount of hidden apps is hidden (`display: none`), so I can't open the dialog normally

fixed

> * It is not apparent that the number is clickable unless you hover over it, some better indication is required (this should also resolve your first point)

I've been trying to create a separate section for this instead of reusing the stats styles, but haven't had time to complete it. In case a new version is imminent, maybe go with this and improve it later?

> * The texts "View on ITAD" and "View on SteamDB" should be replaced by their respective icons

fixed

> * Should the link to the community page go to the homepage or to the discussions?

Discussions is the more useful option, I think.

> * Ad 3: I think it's the other way around, since you actually free up space by removing the row

I guess it's fine after all.
